integration/query_test.go

```diff
@@ -473,6 +474,10 @@ func TestDotNotation(t *testing.T) {
 			filter:      bson.D{{"wsx.0.edc.0.rfv", int32(1)}},
 			expectedIDs: []any{"document-deeply-nested"},
 		},
+		"FieldID": {
+			filter:      bson.D{{"_id.foo", "bar"}},
+			expectedIDs: []any{bson.D{{"foo", "bar"}}},
```

Comment:
Is it possible to add this to compat test? We prefer to compare result against compat unless we have reason why not to.

Reply:
We might need to add the _id field into the tests, but from my understanding for all of them we can only set "v" key, right?

Reply:
You are right looks like the setup only allows _id to be comparable https://github.com/FerretDB/FerretDB/blob/main/integration/shareddata/shareddata.go#L149. So cannot be bson for now.